External/FEXCore/Source/Interface/Core/ArchHelpers/Arm64Emitter.cpp

```diff
@@ -0,0 +1,219 @@
+#include "Interface/Core/ArchHelpers/Arm64Emitter.h"
+
+#include <FEXCore/Utils/LogManager.h>
+#include <FEXCore/Core/CoreState.h>
+
+#include "aarch64/cpu-aarch64.h"
+
+namespace FEXCore::CPU {
+#define STATE x28
+
+Arm64Emitter::Arm64Emitter() : vixl::aarch64::Assembler(vixl::aarch64::PositionDependentCode) {
```

Comment:
The intention here is that Assembler will allocate an empty codebuffer, which arm64jit can replace later.